packages/angular-cli/models/webpack-build-common.ts

```diff
@@ -33,7 +33,8 @@ export function getWebpackCommonConfig(
   return {
     devtool: 'source-map',
     resolve: {
-      extensions: ['.ts', '.js']
+      extensions: ['.ts', '.js'],
+      modules: [ path.join(projectRoot, 'node_modules') ]
```

Comment:
See https://github.com/angular/angular-cli/pull/2291#issuecomment-252457681 for details.